samples/application.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   namespace: sample
   name: sample-one
 spec:
-  image: "image"
+  image: "quay.io/brancz/prometheus-example-app:v0.4.0"
```

Comment:
Intentional change?

Reply:
When testing locally, the "image" tag will cause some verbose errors which are somewhat of a nuisance. I could've chosen a "simpler" image, but we already know that this one works so me and Even used that for testing.